pages/common/adb-reboot.md

```diff
@@ -0,0 +1,20 @@
+# adb reboot
+
+> Android Debug Bridge Install: reboot an Android emulator instance or connected Android devices.
+> More information: <https://developer.android.com/tools/adb>.
```

Comment:
This page doesn't seem to contain any instructions on how to use adb reboot. Lets use manned instead.
```suggestion
> More information: <https://manned.org/adb>.
```

Reply:
It needs to be updated on other pages then.
https://github.com/tldr-pages/tldr/blob/main/pages/common/adb-shell.md

Reply:
yeah sure